custom-vu/src/main/kotlin/jces1209/vu/page/issue/CloudProjectIssueNavigatorPage.kt

```diff
@@ -0,0 +1,46 @@
+package jces1209.vu.page.issue
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import jces1209.vu.page.issue.AbstractProjectIssueNavigatorPage
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions
+import java.time.Duration
+
+class CloudProjectIssueNavigatorPage(
+    private val driver: WebDriver
+) : AbstractProjectIssueNavigatorPage {
+    override fun openProjectByIndex(numberOfProject: String) {
+        driver
+            .wait(ExpectedConditions
+                .elementToBeClickable(By.xpath("(//*[@data-testid='Content']//a)[$numberOfProject]")))
+            .click()
+        waitForProjectIssueNavigator()
+    }
+
+    fun waitForProjectIssueNavigator() {
+
+        driver.wait(
+            Duration.ofSeconds(30),
+            ExpectedConditions.or(
+                ExpectedConditions.and(
+                    ExpectedConditions.or(
+                        ExpectedConditions.presenceOfElementLocated(By.cssSelector("ol.issue-list")),
+                        ExpectedConditions.presenceOfElementLocated(By.id("issuetable")),
+                        ExpectedConditions.presenceOfElementLocated(By.id("issue-content"))
+                    ),
+                    ExpectedConditions.or(
+                        ExpectedConditions.presenceOfElementLocated(By.id("jira-issue-header")),
+                        ExpectedConditions.presenceOfElementLocated(By.id("key-val"))
+                    ),
+                    ExpectedConditions.or(
+                        ExpectedConditions.presenceOfElementLocated(By.id("new-issue-body-container")),
+                        ExpectedConditions.presenceOfElementLocated(By.className("issue-body-content"))
+                    )
+                ),
+                ExpectedConditions.presenceOfElementLocated(By.className("no-results-hint")) // TODO is it too optimistic like in SearchServerFilter.waitForIssueNavigator ?
```

Comment:
This looks familiar. Let's create/reuse an abstraction.

- [ ] reuse issue-navigator code rather than copy-paste

Reply:
added new abstract class. Name and refactor IssueNavigator added into technical debt.